examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

```diff
@@ -73,4 +75,16 @@ public ArmeriaClientConfigurator armeriaClientConfigurator(ClientFactory clientF
             builder.factory(clientFactory);
         };
     }
+
+    /**
+     * Returns an {@link AnnotatedServiceRegistrationBean} which will be used to add an annotated HTTP
+     * service object to a {@link ServerBuilder}. If you want to only use Spring controllers, you do not
+     * need to create this bean.
+     */
+    @Bean
+    public AnnotatedServiceRegistrationBean annotatedServiceRegistrationBean(HelloAnnotatedService service) {
+        return new AnnotatedServiceRegistrationBean().setServiceName("hello")
```

Comment:
How about just adding the annotated service in `armeriaServerConfigurator` programmatically? We may want to remove/deprecate `*RegistrationBean` in favor of annotation-based scanning, like we discussed before.